bluebottle/payments/services.py

```diff
@@ -1,6 +1,6 @@
 from bluebottle.payments.exception import PaymentException
 import re
-from django.conf import settings
+from bluebottle.clients import properties
```

Comment:
settings aren't thread safe so modifying it one one request may affect a parallel request. And modifying settings in general is tricky anyway - you need to be careful to restore previous values, (e.g. exceptions) etc.

properties will default back to settings if a value is not found in the client's properties. But you *will* need to access them through bluebottle.clients.settings - there's no (sane) way to support settings in a multi tenant setup